solr-search/src/main/java/edu/unc/lib/dl/search/solr/model/SimpleIdRequest.java

```diff
@@ -18,56 +18,55 @@
 import java.util.List;
 
 import edu.unc.lib.dl.acl.util.AccessGroupSet;
+import edu.unc.lib.dl.fcrepo4.PIDs;
+import edu.unc.lib.dl.fedora.PID;
 
 /**
  * Request object for a single ID along with access restrictions and requested result data.
  * @author bbpennel
  */
 public class SimpleIdRequest {
-    protected String id;
+    protected final PID pid;
     protected List<String> resultFields;
-    protected AccessGroupSet accessGroups;
+    protected final AccessGroupSet accessGroups;
 
-    public SimpleIdRequest(String id) {
-        this.id = id;
+    public SimpleIdRequest(String id, AccessGroupSet accessGroups) {
+        this(id, null, accessGroups);
     }
 
-    public SimpleIdRequest(List<String> resultFields, AccessGroupSet accessGroups) {
-        this.accessGroups = accessGroups;
-        this.resultFields = resultFields;
+    public SimpleIdRequest(PID pid, AccessGroupSet accessGroups) {
+        this(pid, null, accessGroups);
     }
 
-    public SimpleIdRequest(String id, List<String> resultFields) {
-        this.id = id;
-        this.resultFields = resultFields;
+    protected SimpleIdRequest(List<String> resultFields, AccessGroupSet accessGroups) {
```

Comment:
why is this protected?

Reply:
It was only being used by a subclass. That's silly though, so I went ahead and deleted this constructor and used a different super constructor in the child class instead. Thanks for making me rethink that